### PR TITLE
Initial server setup

### PR DESCRIPTION
**Pull request by RQ16**
#9

### CODE REVIEW

File: src/routes/home.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = (req, res) => {
```
Comment (m4v15) [inline src/routes/home.js]: From what I've seen, more standard express layout is to call these controllers, not routes - https://www.terlici.com/2014/08/25/best-practices-express-structure.html

Comment (m4v15) [reply src/routes/home.js]: so the router.js file is a router which you telling it what to do on the route but this home.js file isn't a route necessarily it's a controller(or handler, but controller seems to be the term used in express) which your router.js calls on a route ('/')

File: src/routes/router.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const router = require('express').Router();
+
+router.get('/', require('./home.js'))
```
Comment (m4v15) [inline src/routes/router.js]: I think this is fine for the home route but for future routes possibly:

```js
router.route('places')
      .get(controllerhere)
      .post(contollerher)
```

Looks a lot better/is standard

**Review by m4v15 — CHANGES_REQUESTED** [review]

**Comment by m4v15** [comment]
Sorry just a minor change to the folder name and potentially future route structure, otherwise it looks 👌  🥂

File: src/routes/home.js (hunk 1 above)
Comment (m4v15) [reply src/routes/home.js]: let me know if that didn't make sense it's a bit of a ramble

**Comment by m4v15** [comment]
assigning you @des-des as well as I think you said at one point you might want to look over the first few PRs and this one is small

File: src/routes/home.js (hunk 1 above)
Comment (des-des) [reply src/routes/home.js]: what mavis said

**Review by des-des — APPROVED** [review]

File: tests/test.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const supertest = require('supertest');
+const tape = require('tape');
+const app = require('../src/app.js');
+
+tape('GET /', (t) => {
+  supertest(app)
+    .get('/')
+    .expect(200)
+    .end((err, res) => {
+      t.error(err);
```
Comment (m4v15) [inline tests/test.js]: At the moment this test gives the text `null` which isn't super helpful, could just add a message like:

` t.error(err, 'Error test');`

Also I guess this is just a initial set up, but you're not really testing anything here, just that you don't get an error. 

Could have a check to test the right response is sent back - check that the `res.text` includes something from your template?

**Review by m4v15 — CHANGES_REQUESTED** [review]

File: tests/test.js (hunk 3 above)
Comment (RQ16) [reply tests/test.js]: Since it is an initial setup so I thought there was no need to

Comment (m4v15) [reply tests/test.js]: Yeah I mean it's not a huge deal either way, I just wasn't sure

**Review by m4v15 — APPROVED** [review]

**Pull request merged**